### ISSUE TITLE
[BUG] Next meeting notification not showing in both screens

### ISSUE THREAD

**Issue by pietrobolcato** [pakerwreah/Calendr#169]
Hey there, first of all wanted to say a big big thank you for the amazing work on the app, it's incredibly helpful and well done! greatly appreciate all you do

I have a tiny bug to report: when having two screens, in the one that is not on focus, the next meeting notification is not shown, and there's white space only, eg:

- Focused screen:

![image](https://github.com/pakerwreah/Calendr/assets/3061306/cab0f9e1-89b5-4dad-8897-d77188d27827)

- Out of focus screen:

<img width="1792" alt="image" src="https://github.com/pakerwreah/Calendr/assets/3061306/84f9bf56-23b4-483f-a753-a17c9d220c0f">

Wonder if this was already known, or is it happening to me only? using latest version, thanks a lot! 🙏

**Comment by pakerwreah** [comment]
Thanks! 🙇🏻‍♂️ 

Yeah, this is a known issue, that's because I attach the view outside of the menu bar button.
It's a compromise to not have huge empty spaces before and after the item.
Unfortunately, that's not how Apple wants us to use it.

Even if I fix it, they are not 2 separate elements, so it will expand to a larger size when you switch to a screen without the notch and might not even fit the macbook's menu bar.

TL;DR: too many technical issues to worry about 😅

**Comment by pietrobolcato** [comment]
i see thanks a lot for the answer! no worries it is a tiny detail indeed :) keep up the great work!
